cogs/info.py

```diff
@@ -107,6 +107,17 @@ async def about(self, ctx):
 
         await ctx.send(content=f"ℹ About **{ctx.bot.user}** | **{self.config['version']}**", embed=embed)
 
+    ## Logging messages sent in chat for use with @story{11}
```

Comment:
This Doxygen comment could be formatted like the other comments to keep it consistent in all the files as you have it on line 13.